### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by noahlitvin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,44 +1,60 @@
...
+            id: true,
+            balanceOf: true,
+            token: { symbol: true, name: true }
+        }
+    );
+
+    console.log('Top Balances');
     for (const entry of topBalances) {
-        console.log(`${entry.id}: ${entry.balanceOf.toString(2)}`);
+        console.log(`${entry.id}: ${entry.balanceOf.toLocaleString()}`);
```
Comment (dbeal-eth) [inline README.md]: does `wei` have `toLocaleString`? I dont think so...

File: README.md [hunk 2]
```diff
@@ -50,11 +66,7 @@ const text = cgt.gen({
 fs.writeFileSync(`subgraphQuery.ts`, text);
 ```
 
-## Supported Methods
-
-Currently supported generation methods are `plain` (raw async/await) and `reactquery` (react hooks which query subgraph--suitable for direct usage in frontend code)
```
Comment (dbeal-eth) [inline README.md:55]: how do we explain these different formats without having this secton here?

**Review by dbeal-eth — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (noahlitvin) [reply README.md]: Ah, yeah. Wanted the commas in the output, but will switch back to keep it simple.

File: README.md (hunk 2 above)
Comment (noahlitvin) [reply README.md:55]: I imagined line 8 covered it, but making some changes to make it more explicit.

File: README.md [hunk 3]
```diff
@@ -1,31 +1,47 @@
...
+
+Now you now have a TypeScript file, `subgraph.ts`, which can be used to query data. Review the file for the names of the generated functions.
+
+The code snippet below could be used to display the top 1,500 user balances from a hypothetical subgraph at the example endpoint:
 
 ```
 import { getManyUserBalance } from './subgraph';
 
-const MY_SUBGRAPH_URL = 'https://thegraph...';
+const
```
Comment (dbeal-eth) [inline README.md]: interesting line break

**Review by dbeal-eth — APPROVED** [review]
you didn't have to do this, thanks!

**Pull request merged**